### DISCUSSION TITLE
Font Request

### DISCUSSION THREAD

**Discussion by TheArchitectEngineer** [Project-Prism/Prism-OS#27] [Q&A] [answered]
Hi 

What is the font used on that Screenshot ? 

![prism-2021-10-11-14-23-01](https://user-images.githubusercontent.com/11510474/150969012-4691429e-7741-4239-9948-7500cb4d2e9a.png)

Advance Thanks ;)

**Comment by terminal-cs** [comment, marked as answer]
I am not sure what the font is called, all I know is that it isn't a regular ttf font. You can try to convert it yourself if you want to try, shouldn't be too hard in theory
